desktop/cmp/tab/container/TabContainerModel.js

```diff
@@ -136,6 +156,11 @@ export class TabContainerModel {
                     });
 
                     if (activateTab) {
+                        if (activateTab.disabled) {
+                            console.warn(`Attempted to route to disabled tab ${activateTab.id}`);
```

Comment:
Think you could skip this warning.

Reply:
Should I also remove the warning in `activateTab`?